### ISSUE TITLE
Label improvements

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1164]
### Issue 1

Linked labels have broken hover styles:
![image](https://user-images.githubusercontent.com/756393/76981183-19c76280-6932-11ea-8959-62c9a4d4600a.png)
See https://pulsar.docs.jadu.net/components/label#linked-labels for examples.

### Issue 2
Removable labels cannot be accessed by the keyboard due to the remove control being a link with no `href`. This should be swapped to a button.
